config/constants/routes.js

```diff
@@ -0,0 +1 @@
+exports.ADMIN_USERS = '/admin/users';
```

Comment:
Why is this?

Reply:
I see where you use it. I think it won't be needed.

Reply:
I put it because maybe in the future the path could be changed, so if this changes, we don't need to change in all files where it is mentioned. Only for that

Reply:
It is a good approach, but for consistency there are some changes you will have to do in order to keep it 😛 :
1. Use this in the routes file
2. Use this in docs file
3. Use this for every other route

Reply:
Yes. But it is approved?

Reply:
It depends on the projects. Usually it is not used, but it could be. The main conflict I see is the parametrization of the routes, which may not be the same in every part of the code, but I suppose it could be mitigated.
So, yes it would be approved, but not as a standard practice.